surfstudio/ios-kinescope-sdk
# Feature/kin 48 control panel skeleton

## Changes

- Init configurable controls of control panel 
- Add layout of control panel
- Add input/output protocols for control panel
- ...

## Open questions

- ...

## How to test?

- Open Video View Controller
- Check stubs for control panel views
